cypress-tests/cypress/e2e/spec/Payment/00019-MandatesUsingPMID.cy.js

```diff
@@ -26,6 +26,10 @@ describe("Card - Mandates using Payment Method Id flow test", () => {
         }
       });
 
+      it("customer-create-call-test", () => {
+        cy.createCustomerCallTest(fixtures.customerCreateBody, globalState);
+      });
+
```

Comment:
Please verify that it does not affect other connectors. Saying so because you've touched the spec files and it affects all connectors.